packages/ui/src/utils/channelUtils.tsx

```diff
@@ -13,15 +14,15 @@ export function getChannelMemberName(
   if (disableNicknames) {
     return member.contactId;
   }
-  return member.contact?.nickname ?? member.contactId;
+  return member.contact?.nickname ?? formatUserId(member.contactId)?.display;
```

Comment:
we've run into null coalesce issues in these kinds of comparisons if nickname is set but it's an empty string — can we make sure we defend against that?